chidinma_api_page/api_script.js

```diff
@@ -0,0 +1,10 @@
+function getData(){
+    
+    let a = document.getElementById('input').value 
+
+    fetch('https://jsonplaceholder.typicode.com/users/' + a)
+    .then(response => response.json())
+    .then(json => console.log(json))
```

Comment:
Looks good so far. This is the part that will need to be updated. Instead of writing the json to the console log, you should update the output area of your page. So replace the console log statement with a statement that will update the output element with the json